### ISSUE TITLE
Tries to send message to deleted users

### ISSUE THREAD

**Issue by t-schroeder** [lsuonline/lsuce-block_quickmail#20]
We noticed one message with two pending recipients. Those are both users that have beend deleted for 3 months already. Deleted users have some hash in their email address field. So of course delivery of the message (via email) fails. Quickmail tries to send the message every minute to these two recipients. Everytime it does that it updates the message's "sent_at" field in the DB. Therefore this message always appears at the top of sent messages for the author, even though it was originally sent 3 months ago.

In conclusion quickmail should check if the recipients are deleted. If they have been it should probably list them under a new section "Deleted recipients". Ideally it would also generate a notification telling the message author about this. Otherwise they won't notice that some recipients couldn't be reached because they were deleted.
